examples/kitchen-sink/.env.sample

```diff
@@ -0,0 +1,2 @@
+MONGO_URI="mongodb+srv://<username>:<password>@cluster0.wffq71a.mongodb.net/?retryWrites=true&w=majority"
```

Comment:
does this need to go in the new starter?

Reply:
removed